lib/publiccloud/ssh_interactive.pm

```diff
@@ -125,4 +125,41 @@ sub select_host_console {
     assert_script_run("true", fail_message => "host console is broken");    # basic health check
 }
 
+sub prepare_ssh_tunnel {
```

Comment:
I think it make sense . @grisu48 , @pdostal  WDYT ?

Reply:
A prima vista, yes. I would like to see some verification runs on our maintenance test runs though.

Reply:
Hi @grisu48 & @asmorodskyi,
I have revised code according to you comments and did VR, please help to check again, thanks :)
VR-1 for `mr_test`:
   passed as expected: https://openqaworker15.qa.suse.cz/tests/67499#
   failed as expected: http://openqaworker15.qa.suse.cz/tests/67504#
  
VR-2 for `qem` test cases
   publiccloud_nfs passed: http://openqa.suse.de/tests/9961054
   publiccloud_smoketests_gen1 passed: http://openqa.suse.de/tests/9961043
   
VR-3 for `post fail cleanup` on purpose
    sles4sap_gnome_saptune_delete_rename: http://openqaworker15.qa.suse.cz/t67504